datastore/google/cloud/datastore/_http.py

```diff
@@ -451,25 +434,6 @@ def rollback(self, project, transaction_id):
         # Response is empty (i.e. no fields) but we return it anyway.
         return self._datastore_api.rollback(project, request)
 
-    def allocate_ids(self, project, key_pbs):
-        """Obtain backend-generated IDs for a set of keys.
-
-        Maps the ``DatastoreService.AllocateIds`` protobuf RPC.
-
-        :type project: str
-        :param project: The project to which the transaction belongs.
-
-        :type key_pbs: list of
-                       :class:`.entity_pb2.Key`
-        :param key_pbs: The keys for which the backend should allocate IDs.
-
-        :rtype: :class:`.datastore_pb2.AllocateIdsResponse`
-        :returns: The protobuf response from an allocate IDs request.
-        """
-        request = _datastore_pb2.AllocateIdsRequest()
-        _add_keys_to_request(request.keys, key_pbs)
-        return self._datastore_api.allocate_ids(project, request)
-
 
 class HTTPDatastoreAPI(object):
```

Comment:
`_DatastoreAPIOverHttp` and `Connection` are going to the junkyard once this process is finished.